Fixed broken link (404) (#1243)
ipfs/ipfs-docs: docs/how-to/run-ipfs-inside-docker.md

```

:::tip Use in custom images

> See `gateway` example at[github.com/ipfs-shipyard/go-ipfs-docker-examples](https://github.com/ipfs-shipyard/go-ipfs-docker-examples)

Reviewer: ```suggestion
See the `gateway` example on the [go-ipfs-docker examples repository](https://github.com/ipfs-shipyard/go-ipfs-docker-examples)
```
